workers/genotype_extractor.py

```diff
@@ -21,6 +21,11 @@
 
 @worker.task(bind=True)
 def extract(self, vcf_id):
+    register_running_task(self, vcf_id)
+    return _extract(vcf_id)
```

Comment:
I can do that, but I'd still rather have an `_extract` method: in this case I think the underscore is warranted; the primary export of this module is the worker task, but the worker task is doing something somewhat different than `_extract` alone (namely registering itself, plus everything the decorator is doing [wrapping it in a class specific to Celery, I believe?]).